src/components/AnnouncementCarousel/AnnouncementCarousel.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import Slider from 'react-slick'
+import styles from './AnnouncementCarousel.module.scss'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import AnnouncementInfo from 'components/AnnouncementInfo/AnnouncementInfo'
+import 'slick-carousel/slick/slick.css'
+import 'slick-carousel/slick/slick-theme.css'
+import { AnnouncementRecord } from 'types'
+
+const NextArrow = ({ onClick }: any) => {
```

Comment:
Yeah I remember having this conversation with Suz. I'm sure there is a way to define a type for this, but my Typescript isn't good enough to figure it out. I just added in a line for the linter to ignore the warnings for now. Maybe once you get up to speed on Typescript, we could loop back to this and pair on it?

Reply:
Sounds good. I'm fine if you leave the ignore line out since that will remind me to follow up on it. But either way not something I want to block this PR on.